pom.xml

```diff
@@ -28,10 +28,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <scala.maven.version>3.2.2</scala.maven.version>
-        <scala.binary.version>2.11</scala.binary.version>
```

Comment:
Removed because cross-compilation plugin requires / recommends they are not defaulted and only set through Profiles.

I was able to select a Profile inside IntelliJ and the project compiled in my IDE.